Wabbajack.App.Wpf/View Models/Installers/InstallerVM.cs

```diff
@@ -325,10 +338,43 @@ private IEnumerable<ErrorResponse> Validate()
 
         if (KnownFolders.IsInSpecialFolder(installPath) || KnownFolders.IsInSpecialFolder(downloadPath))
         {
-            yield return ErrorResponse.Fail("Can't install a modlist into Windows protected locations - such as Downloads, Documents etc");
+            yield return ErrorResponse.Fail("Can't install a modlist into Windows protected locations - such as Downloads,Documents etc, please make a new folder for the modlist.");
```

Comment:
This may not be explicit enough, since users may just make a new folder inside a protected location since that technically fulfills the requirements laid out here (Not directly inside a protected folder, and in a new folder). 

I'd recommend adding a suggestion as already done for the launch path.